extensions/lib/erb_helper.rb

```diff
@@ -0,0 +1,19 @@
+require "web_console/fake_middleware"
```

Comment:
Instead of executing this through an external process, let's make this a library and use it as Ruby code in the Rakefile. I'm kinda troubled about where this should live. We can put it in `lib/web_console/testing/erb_helper` for now.